JASP-R-Interface/jaspResults/src/jaspContainer.h

```diff
@@ -43,7 +43,7 @@ class jaspContainer : public jaspObject
 	void		checkDependenciesChildren(Json::Value currentOptions) override;
 
 	void		completeChildren();
-
+	void		setError(bool error);
```

Comment:
Because you override the virtual function on jaspObject you should add `override`, aka:
`void		setError(bool error) override;` otherwise you hide the original function and this might have unintended consequences.

Reply:
I guess this should be the start of a request for changes.

Reply:
Good one! no idea how this stuff with `virtual` and `override` really works.